.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
This looks like it's set up to every time a branch is pushed regardless of if a PR is created. I usually set it up just to run on PRs since I'll sometimes commit things or create test branches which I never expect to pass a build. I don't think it's a problem the way it's set up though, just a preference.

Reply:
I do sometimes check build statuses before making PRs, and some other members of Research IT have found it helpful too, so I'll keep the two triggers. Thanks for the input though!